art/attacks/__init__.py

```diff
@@ -2,4 +2,4 @@
 Module providing adversarial attacks under a common interface.
 """
 from art.attacks.attack import Attack, EvasionAttack, PoisoningAttack, PoisoningAttackBlackBox, PoisoningAttackWhiteBox
-from art.attacks.attack import ExtractionAttack, InferenceAttack, AttributeInferenceAttack
+from art.attacks.attack import ExtractionAttack, InferenceAttack, AttributeInferenceAttack
```

Comment:
Please don't remove the newline character at the end of this line.
```suggestion
from art.attacks.attack import ExtractionAttack, InferenceAttack, AttributeInferenceAttack

```